### ISSUE TITLE
Login times are not set when auto login after password reset is on

### ISSUE THREAD

**Issue by mauritsvanrees** [plone/Products.CMFPlone#3713]
### What I did:

- Plone coredev 5.2, Python 2. Seen on released Plone 5.2.10 with Python 3 as well. Likely Plone 6 as well, but I need to check.
- Create a Plone site.
- Configure the mail settings, or make sure Products.PrintingMailHost is used.
- Go to the Security Settings.
- Check that auto login after password reset is turned on. This is the default in 5.2.
- Enable open registration (optional, but makes it easier to check).
- As anonymous user register an account.
- You get a password reset mail. Follow the link.
- Set your password.
- You are now logged in.

### What I expect to happen:

The `login_time` and `last_login_time` properties are set to today.

### What actually happened:

The `login_time` and `last_login_time` properties are set to January 1, 2000.

You cannot easily see this information anywhere. I edited the main template:

```
--- a/Products/CMFPlone/browser/templates/main_template.pt
+++ b/Products/CMFPlone/browser/templates/main_template.pt
@@ -79,6 +79,11 @@
         <metal:slot define-slot="body">
 
         <article id="content">
+          <div tal:define="member context/@@plone_portal_state/member">
+            Login time: ${python:member.getProperty('login_time', 'no login_time')}
+            <br />
+            Last login time: ${python:member.getProperty('last_login_time', 'no last_login_time')}
+          </div>
 
           <metal:bodytext define-slot="main">
```

Note that this snippet also works for anonymous users: you will see 'no login_time'.

This bug may not sound like a big deal. But when you have custom code that weekly goes through all members, and removes everyone without a login time when they do not have a pending password reset request, then you will remove innocent users... I managed to restore them from a backup.

Workaround: disable the auto login.

**Comment by mauritsvanrees** [comment]
Plone 6.0.0 has the same problem.

**Comment by mauritsvanrees** [comment]
My fixes have been merged, and will be released in Plone 5.2.11 and 6.0.1, expected within a week, or two.
